### ISSUE TITLE
Result on test sets

### ISSUE THREAD

**Issue by David9591** [zju3dv/snake#173]
Dear author:
First of all, thank you for your excellent work, but I have a small question:
After I run the following command:
`python run.py --type evaluate --cfg_file configs/city_rcnn_snake.yaml test.dataset CityscapesVal`
What I get is Table3:AP[val]37.4 in your paper，it is the result on Cityscapes Val.
But how do I get the results of  test set(remaining columns)?I tried to run the following command:
`python run.py --type evaluate --cfg_file configs/city_rcnn_snake.yaml test.dataset CityscapesTest`,
It's wrong.Could you help me solve this problem or give me some advices?
Best wishes!

**Comment by pengsida** [comment]
To get the results of test set, you need to submit the segmentation results to the official site of Cityscapes.

**Comment by David9591** [comment]
> To get the results of test set, you need to submit the segmentation results to the official site of Cityscapes.

Yes, I checked the description of the Cityscapes dataset and found that TEST was not publicly marked.The further question I would like to ask is how to test the TEST dataset through the code, as I wrote it?
`python run.py --type evaluate --cfg_file configs/city_rcnn_snake.yaml test.dataset CityscapesTest`
Best wishes!

**Comment by pengsida** [comment]
Yes.
